views/assets/javascripts/qor/qor-action.js

```diff
@@ -345,7 +345,7 @@
 
     QorAction.TEMPLATE_LOADING = `<div class="qor-body__loading">
                                         <div class="mdl-dialog-bg"></div>
-                                        <div><div class="mdl-spinner mdl-js-spinner is-active qor-layout__bottomsheet-spinner"></div></div>
+                                        <div class="loading relative"><div class="loading-bar"></div><div class="loading-bar"></div><div class="loading-bar"></div><div class="loading-bar"></div></div>
```

Comment:
It seems like there's a lot of room for improvement on this approach for a loading state. 
Instead of having to put a div with some classes, and then other 4 divs inside with other classes, each time, we could use a gif/svg, or a custom component that encapsulates this stuff.